smaily-connect.php

```diff
@@ -1,15 +1,17 @@
 <?php
 /*
  * Author URI:        https://smaily.com
- * Author:            Sendsmaily LLC
- * Description:       Smaily integration plugin that includes WooCommerce and Contact Form 7 integrations.
+ * Author:            Smaily
+ * Description:       Smaily integration plugin that includes WooCommerce, Elementor and Contact Form 7 integrations.
  * Domain Path:       /languages
  * License URI:       https://www.gnu.org/licenses/gpl-3.0.en.html
  * License:           GPL-3.0+
+ * Requires at least: 6.0
+ * Requires PHP:      7.0
  * Plugin Name:       Smaily Connect
- * Plugin URI:        https://github.com/sendsmaily/smaily-wordpress-plugin
+ * Plugin URI:        https://smaily.com/help/user-manual/smaily-connect-for-wordpress/
  * Text Domain:       smaily-connect
- * Version:           1.0.0
+ * Version:           1.1.0
 */
```

Comment:
I have changed the author to Smaily, this aligns better with the domain. Also, the plugin URI directs to the tutorial rather than the developer resource. I think that developers find the resource anyway, so it is better to show a link to the resources for people who are looking for functionality and feature explanations.